waspc/data/Generator/templates/server/src/routes/auth/login.js

```diff
@@ -12,13 +12,13 @@ export default handleRejection(async (req, res) => {
   const context = {}
 
   // Try to fetch user with the given email.
-  const {= userEntityLower =} = await prisma.{= userEntityLower =}.findUnique({ where: { email: args.email.toLowerCase() } })
+  const user = await prisma.{= userEntityLower =}.findUnique({ where: { email: args.email.toLowerCase() } })
   if (!user) {
```

Comment:
AHa, the problem is in the line 16!
HOw is it that you decided to convert everything to `user`, and not this one `user` to `{= userEntityLower =}`?
The benefit was that we would name the variable same way they named their model.
But truth is, this was more interesting when we thought they will be ejecting often. And at the end, it probably makes everything just complicated and harder to read when using `userENtityLower` all around, so I guess that is the reason?

Reply:
Yep, that was the one! I decided to convert all usages for three main reasons:
- `auth.js` used this pattern before: https://github.com/wasp-lang/wasp/blob/main/waspc/data/Generator/templates/server/src/core/auth.js#L41-L51
- In this case, as you noted, I thought it improved readability even in the ejection case since `user` would still be a good generic variable name here.
- In general, I think we should prefer optimizing for our own DX over ejection. I would much prefer to reference that variable throughout the file vs a templated name. In addition, my hope is we can somehow start to encapsulate all templated variables as config inputs and leave the core files as pure JS over time.

However, it is always good to keep ejection in mind. But in this case, I thought those positives warranted the change.

Reply:
Sounds good and I think you made a good choice!